week3OptionalChallenge.js

```diff
@@ -4,7 +4,24 @@ var number=['1','3','5','8','10'];
 var num1=[];
 var missingNum;
 function findMissingNumbers(numArray){
-    
+    if(numArray[0]>1 || numArray[numArray-1]<1){
```

Comment:
What are you trying to achieve here?
`numArray - 1` doesn't look goo to me since you are subtracting a number from an array. Can you change it

Reply:
An alternative approach you can choose is:-
1. Sort the array in ascending order and remove duplicates
2. Get the minimum and maximum value from the array
3. Create a loop through values from the minimum to the maximum while checking whether that value is there in the array.
4. Add it to the array of missing numbers if it doesn't exist

Reply:
i resolved it. hope that's what u meant. thank